### PR TITLE
[FEAT] Simplify some old timestamps

### PR DESCRIPTION
**Pull request by brian-man**

### CODE REVIEW

File: src/lib/flags.ts [hunk 1]
```diff
@@ -8,9 +8,7 @@ const defaultFeatureFlag = ({ inventory }: GameState) =>
 const testnetFeatureFlag = () => CONFIG.NETWORK === "amoy";
 
 const clashOfFactionsFeatureFlag = () => {
-  if (testnetFeatureFlag()) return true;
-
-  return Date.now() > new Date("2024-05-01T00:00:00Z").getTime();
+  return true;
```
Comment (spencerdezartsmith) [inline src/lib/flags.ts:11]: We should actually just remove these old feature flags.

File: src/lib/flags.ts [hunk 2]
```diff
@@ -75,12 +73,7 @@ const featureFlags: Record<FeatureName, FeatureFlag> = {
   },
   FACTION_HOUSE: defaultFeatureFlag,
   EASTER: (game) => {
```
Comment (spencerdezartsmith) [inline src/lib/flags.ts:75]: This one makes sense to keep as easter rolls around each year 🐰

**Review by spencerdezartsmith — CHANGES_REQUESTED** [review]
I love a good tidy up. I think we can remove the old season flag now.

File: src/lib/flags.ts (hunk 1 above)
Comment (brian-man) [reply src/lib/flags.ts:11]: I was going to do that in a separate PR.

**Review by spencerdezartsmith — APPROVED** [review]
LGTM

**Pull request merged**